debezium-ddl-parser/src/main/java/io/debezium/antlr/ProxyParseTreeListenerUtil.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+/*
```

Comment:
Hey @kucharo2, why is that licensed under BSD 3-clause?

Reply:
I was not sure about the licence header in this one. The implementation is taken from ANTLR repo, and they are using BSD license (the same is with CaseChangingCharStream).